test/pool.js

```diff
@@ -194,7 +194,7 @@ describe('Pool', function() {
       this.emit('disconnect', this, {});
     });
 
-    var pool = new Pool(null, { size: 1 });
+    var pool = new Pool(null, { dnsSeed: false });
```

Comment:
Check the tests, if there is only one addr, the max it can go is one, unless other addrs are added.